### PR TITLE
Update co-authoring-overview for Issue #960

### PR DESCRIPTION
**Pull request by ojrb**
I modify the documentation about the major versioning enablement by default as requested, this PR solve the issue #960 @scanum Please review and Merge !

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointServer/governance/co-authoring-overview.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ Unlike Word and PowerPoint, OneNote stores version information within the file i
   
 - Do not enable minor versioning. By default, minor versioning is not enabled in SharePoint Server.
     
-- If major versioning is enabled, set a reasonable maximum number of versions to store. By default, major versioning is not enabled in SharePoint Server.
+- Major versioning is enabled in SharePoint Server by default, set a reasonable maximum number of versions to store.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/governance/co-authoring-overview.md]:
```suggestion
- Major versioning is enabled in SharePoint Server by default. Set a reasonable maximum number of versions to store.  
```

**Comment by ojrb** [comment]
Hi @scanum I committed the suggestions of @JohanFreelancer9 !

**Comment by scanum** [comment]
@tomresing Hi Tom, changes look correct. Could you please check and merge? Thanks!

**Pull request merged**